go/pkg/log/server/property_test.go

```diff
@@ -86,99 +243,185 @@ func (suite *LogServerTestSuite) TestRecordLogDb_PushLogs() {
 				if int32(len(records)) != r.RecordCount {
 					t.Fatal("record count mismatch", len(records), r.RecordCount)
 				}
-				suite.model.CollectionData[c] = append(suite.model.CollectionData[c], records...)
 			},
-			"getAllCollectionsToCompact": func(t *rapid.T) {
+			"compaction": func(t *rapid.T) {
 				result, err := suite.logServer.GetAllCollectionInfoToCompact(ctx, &logservicepb.GetAllCollectionInfoToCompactRequest{})
 				assert.NoError(suite.t, err)
+
 				for _, collection := range result.AllCollectionInfo {
 					id, err := types.Parse(collection.CollectionId)
 					if err != nil {
 						t.Fatal(err)
 					}
-					compactionOffset := rapid.Int64Range(suite.model.CollectionCompactionOffset[id], int64(len(suite.model.CollectionData))).Draw(t, "new_position")
+					enumerationOffset := suite.model.CollectionEnumerationOffset[id]
```

Comment:
Can you add some comments indicating add to the model and add to the test suite? Also, also, it would be great to introduce some methods in the model for push_log, compaction, etc.